views.py

```diff
@@ -92,18 +50,13 @@ def add_new_user():
     user_data = request.form
     app.logger.info(f'User data: {user_data}')
 
-    email = request.form.get('email')
-    username = request.form.get('username')
-    password = request.form.get('password')
-
-    new_user = User(email=email, 
-                    username=username, 
-                    password=password)
+    # As long as the form fields exactly match the fields on your User model, this works.
+    new_user = User(**request.form)
```

Comment:
So the goal overall is for all your object interfaces to be consistent. If the `User` model has username, email and password, then it's corresponding form should have exactly the same. You wouldn't name the fields user_name for example, because that tiny inconsistency makes it difficult to handle the user *for the whole app*. 

Here, assuming my form matches my model,I can now make creating a new user as simple as this line of code.